packages/kmod-5.10-nvidia/kmod-5.10-nvidia.spec

```diff
@@ -148,10 +149,15 @@ install -m 755 nvidia-smi %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{te
 install -m 755 nvidia-debugdump %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{tesla_470}
 install -m 755 nvidia-cuda-mps-control %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{tesla_470}
 install -m 755 nvidia-cuda-mps-server %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{tesla_470}
+install -m 755 nvidia-persistenced %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{tesla_470}
```

Comment:
I thought about installing `nvidia-persistenced` in `%{_cross_bindir}`, since it is meant to be a system service. The problem with this approach is that if in the future NVIDIA ships other `run` archives different than the `tesla` archive and we have to include it, there might be two versions of `nvidia-persistenced` that would have to be shipped.

I think we can keep it as it is, and going forward, we could do some guessing at runtime based on the driver that was loaded to override the path to `nvidia-persistenced` using systemd drop-ins.